### ISSUE TITLE
Warning: "my" $store in datastore_volumes_info.pm

### ISSUE THREAD

**Issue by realredknight** [BaldMansMojo/check_vmware_esx#163]
Hi, 

with the the Update to 1.22 I get 

> "my" variable $store masks earlier declaration in same scope at /usr/lib/nagios/plugins/modules/datastore_volumes_info.pm line 32.

as first line of the results. 

The check is running on Ubuntu 18.04 with perl 5.26.1

**Comment by fireheadman** [comment]
Also getting this on redhat....
```
[user@icinga01 check_vmware_esx 3228]$ ./check_vmware_esx.pl -H esxihost01 -f /etc/icinga2/authfiles/vcenter-auth -S cpu -s usage
"my" variable $store masks earlier declaration in same scope at modules/datastore_volumes_info.pm line 32.
```

Not really wanting to go fishing in the modules to find this as this is a very complex package... please assist.

**Comment by BaldMansMojo** [comment]
Fixed. It was only a duplicated line.

**Comment by fireheadman** [comment]
UPDATE:    
While I did hit this issue last night.... I reverted my snapshot on my icinga master and reinstalled the SDK.   This time I did not get the message.

**Comment by BaldMansMojo** [comment]
As mentioned - it was a duplicate definition. But normally this warning will go to STDERR and therefore not be harm the output in the monitor system.
